### PR TITLE
Insertion operation added

### PR DESCRIPTION
**Pull request by sheetal2605**

### CODE REVIEW

File: GitHubMainApplication/Insertion.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{
+    public static class Insertion
+    {
+        public static void AddElement(int[] array, int count, int insertElement, int index)
+        {
+            count++;
+            int elementAtIndex = array[index];
+            array[index] = insertElement;
+            int temporary;
+            for (int iteration = index+1; iteration < count; iteration++)
```
Comment (tshet-eg) [inline GitHubMainApplication/Insertion.cs]: can you comment on what this does

File: GitHubMainApplication/Insertion.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            {
+                temporary = array[iteration];
+                array[iteration] = elementAtIndex;
+                elementAtIndex = temporary;
+               
+            }
+            for(int i=0;i< count; i++)
```
Comment (tshet-eg) [inline GitHubMainApplication/Insertion.cs]: change the variable name

**Review by rchanRakshith — COMMENTED** [review]
everything is fine

**Review by Manish-EG — COMMENTED** [review]
It is ok

**Pull request merged**